OIPA/api/export/tests/test_full_activity.py

```diff
@@ -208,6 +210,7 @@ def test_create_activity(self):
 
 
         xml = iati_activities(
+                ET.Comment(settings.EXPORT_COMMENT),
```

Comment:
@HatimKH  What's this? Should we set EXPORT_COMMENT in test_settings.py / travis_test_settings.py ?